scripts/build.js

```diff
@@ -30,15 +30,15 @@ const ERROR = chalk.reset.inverse.bold.red( ' ERROR ' );
 /**
  * Babel Configuration
  */
-const babelDefaultConfig = require( '../packages/babel-preset-default' );
+const babelDefaultConfig = require( '../packages/babel-preset-default' )();
```

Comment:
Babel expects functions in presets in 7.x line.

Reply:
Now I see:

> const isTestEnv = api.env() === 'test';
> TypeError: Cannot read property 'env' of undefined

because we need to pass `api` object to this preset ...